tests/conftest.py

```diff
@@ -23,3 +23,50 @@ def es(docker_services):
 @pytest.fixture(scope='session')
 def es_client(es):
     return Elasticsearch([es])
+
+
+@pytest.fixture(scope='session')
+def wiki_es(docker_services):
+    """Ensure that ES is up and responsive."""
+    docker_services.start('wiki_es')
+    port = docker_services.wait_for_service("wiki_es", 9200)
+
+    url = f'http://{docker_services.docker_ip}:{port}'
+
+    for c in app.injector.components:
+        if isinstance(c, SettingsComponent):
```

Comment:
maybe we should add a tests/utils.py with and abstraction to set some settings

Reply:
done, but better can be probably done...